cql3/restrictions/single_table_query_restrictions.cc

```diff
@@ -7,3 +7,71 @@
  */
 
 #include "single_table_query_restrictions.hh"
+#include "cql3/expr/expression.hh"
+#include "cql3/restrictions/clustering_key_restrictions.hh"
+#include "seastar/core/on_internal_error.hh"
+
+namespace cql3 {
+namespace restrictions {
+using namespace expr;
+
+extern logging::logger rlogger;
+
+struct split_where_clause {
+    // Restrictions that only involve partition key columns
+    expression partition;
```

Comment:
Does this mean "all restrictions that involve partition key columns"

or "all restrictions that involve partition key columns and can be part of read_command"?

Reply:
Updated the comment to be more specific.